src/test/TestAccount.cpp

```diff
@@ -24,9 +24,13 @@ TestAccount::loadSequenceNumber() const
 void
 TestAccount::updateSequenceNumber()
```

Comment:
Note that current version of updateSequenceNumber on master breaks the code (because of usage of REQUIRE), so this change is copied from bump sequence PR.